src/group-configurations/experiment-configurations-section/ExperimentForm.jsx

```diff
@@ -0,0 +1,239 @@
+import PropTypes from 'prop-types';
+import { FieldArray, Formik } from 'formik';
+import * as Yup from 'yup';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Alert,
+  ActionRow,
+  Button,
+  Form,
+  OverlayTrigger,
+  Tooltip,
+} from '@edx/paragon';
+import { WarningFilled as WarningFilledIcon } from '@edx/paragon/icons';
+
+import PromptIfDirty from '../../generic/PromptIfDirty';
+import { allGroupNameAreUnique } from './utils';
+import ExperimentFormGroups from './ExperimentFormGroups';
+import messages from './messages';
+
+const ExperimentForm = ({
+  isEditMode,
+  initialValues,
+  isUsedInLocation,
+  onCreateClick,
+  onCancelClick,
+  onDeleteClick,
+  onEditClick,
+}) => {
+  const { formatMessage } = useIntl();
+
+  const validationSchema = Yup.object().shape({
+    id: Yup.number(),
+    name: Yup.string().required(
+      formatMessage(messages.experimentConfigurationNameRequired),
+    ),
+    description: Yup.string(),
+    groups: Yup.array()
+      .of(
+        Yup.object().shape({
+          id: Yup.number(),
+          name: Yup.string().required(
+            formatMessage(messages.experimentConfigurationGroupsNameRequired),
+          ),
+          version: Yup.number(),
+          usage: Yup.array().nullable(true),
+        }),
+      )
+      .required()
+      .min(1, formatMessage(messages.experimentConfigurationGroupsRequired))
+      .test(
+        'unique-group-name-restriction',
+        formatMessage(messages.experimentConfigurationGroupsNameUnique),
+        (values) => allGroupNameAreUnique(values),
+      ),
+    scheme: Yup.string(),
+    version: Yup.number(),
+    parameters: Yup.object(),
+    usage: Yup.array()
+      .of(
+        Yup.object().shape({
+          label: Yup.string(),
+          url: Yup.string(),
+        }),
+      )
+      .nullable(true),
+    active: Yup.bool(),
+  });
+
+  const onSubmitForm = isEditMode ? onEditClick : onCreateClick;
+
+  return (
+    <div
+      className="configuration-card"
+      data-testid="experiment-configuration-form"
+    >
+      <div className="configuration-card-header">
+        <h3>{formatMessage(messages.experimentConfigurationName)}*</h3>
+        {isEditMode && (
+          <span className="text-gray-500">
+            {formatMessage(messages.experimentConfigurationId, {
+              id: initialValues.id,
+            })}
+          </span>
+        )}
+      </div>
+      <Formik
+        initialValues={initialValues}
+        validationSchema={validationSchema}
+        validateOnChange={false}
+        validateOnBlur={false}
+        onSubmit={onSubmitForm}
+      >
+        {({
+          values, errors, dirty, handleChange, handleSubmit,
+        }) => (
+          <>
+            <Form.Group className="mt-3 form-group-configuration" isInvalid={!!errors.name}>
```

Comment:
```suggestion
            <Form.Group className="mt-3 configuration-form-group" isInvalid={!!errors.name}>
```
To stick to the pattern above.

Reply:
renamed